pkg/cloudprovider/providers/gce/kms.go

```diff
@@ -129,19 +134,23 @@ func (gce *GCECloud) getGCPCloudKMSService(config io.Reader) (envelope.Service,
 	return service, nil
 }
 
-// Decrypt decrypts a base64 representation of encrypted bytes.
+// Decrypt decrypts a base64 representation of (key-version-used-to-encrypt + encrypted bytes).
 func (t *gkmsService) Decrypt(data string) ([]byte, error) {
+	dataChunks := strings.SplitN(data, ":", 2)
+	if len(dataChunks) != 2 {
+		return []byte{}, fmt.Errorf("invalid data encountered for decryption: %s. Missing key version", data)
+	}
 	resp, err := t.cloudkmsService.Projects.Locations.KeyRings.CryptoKeys.
 		Decrypt(t.parentName, &cloudkms.DecryptRequest{
-			Ciphertext: data,
+			Ciphertext: dataChunks[1],
 		}).Do()
 	if err != nil {
 		return nil, err
 	}
 	return base64.StdEncoding.DecodeString(resp.Plaintext)
 }
 
-// Encrypt encrypts bytes, and returns base64 representation of the ciphertext.
+// Encrypt encrypts bytes, and returns base64 representation of the (key-version-used-to-encrypt + ciphertext).
```

Comment:
("<key-version-used-to-encrypt>:<encrypted-bytes>")

Reply:
`("<key-version-used-to-encrypt>:<encrypted bytes>")`

Reply:
Fixed both.